### PR TITLE
Mention make targets for checkpatch in documentation.

### PR DESCRIPTION
**Pull request by wamserma**
Signed-off-by: Markus S. Wamser <markus.wamser@mixed-mode.de>

### CODE REVIEW

File: building/gits/optee_test.rst [hunk 1]
```diff
@@ -193,6 +193,15 @@ Coding standards
 ****************
 See :ref:`coding_standards`.
 
+The :ref:`optee_os` repository is required to run the checks. It's location may
+be passed using the `OPTEE_OS_SRC` environment variable:
```
Comment (jforissier) [inline building/gits/optee_test.rst]: Wondering if `OPTEE_OS_PATH` might be more appropriate? Given that we have: https://github.com/OP-TEE/build/blob/3.8.0/common.mk#L14.

Comment (wamserma) [reply building/gits/optee_test.rst]: Yes, why not. Updated both PRs.

**Comment by jbech-linaro** [comment]
Please add Jerome's tag and squash patches where necessary, when done we can merge this.

**Comment by wamserma** [comment]
> Please add Jerome's tag and squash patches where necessary, when done we can merge this.

@jbech-linaro Done (forgot to ping you, though).

**Comment by jbech-linaro** [comment]
Thanks @wamserma , merging!

**Pull request merged**